WooCommerce/Classes/ViewRelated/Dashboard/Settings/POS/POSTabEligibilityChecker.swift

```diff
@@ -30,6 +32,9 @@ final class POSTabEligibilityChecker: POSEntryPointEligibilityCheckerProtocol {
     private let siteSettingService: POSSiteSettingServiceProtocol
     private let appPasswordSupportState: ApplicationPasswordsExperimentState
 
+    /// Local catalog eligibility service - created asynchronously on init
```

Comment:
I looked at how dependencies are created, and I think this could be moved from `POSTabEligibilityChecker` to `POSTabCoordinator`. 

`POSTabCoordinator` is a good choice since:
- It's created at the time of login/site changes
- Responsible for creating/providing POS dependencies
- We pass `eligibilityChecker` into init
- We need dependency to be within `POSTabCoordinator` to check the eligibility

Therefore, we could move the `Task { }` into the init of `POSTabCoordinator` and the tab eligibility checker wouldn't be affected. We could still use `stores` to get credentials and selectedSite.

Reply:
It was there originally. The reason I moved it to the tab eligibility checker was to cleanly check that the store was actually POS eligible in the flow of doing the local catalog eligibility check. I'll look again though, and see if I can do both.